src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -1475,6 +1477,7 @@ BuildSubqueryJobQuery(MultiNode *multiNode)
 	jobQuery->groupClause = groupClauseList;
 	jobQuery->limitOffset = limitOffset;
 	jobQuery->limitCount = limitCount;
+	jobQuery->havingQual = havingQual;
```

Comment:
Hmm, minor issue

```C
jobQuery->hasAggs = contain_agg_clause((Node *) targetList) || contain_agg_clause(havingQual);
```

Although we currently don't support having without group bys,  may it be relevant in the future?